templates/ocs-deployment/pool.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: {{ pool_name | default ('rbd') }}
+  namespace: rook-ceph
```

Comment:
Please add an option to pass parameter here.
Unless it is not allowed, ignore this comment